src/Bindings/ManualBindings.cpp

```diff
@@ -3702,6 +3702,61 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cBookContent_GetPages(lua_State * tolua_S)
+{
+	// cBookContent::GetPages() -> table of strings
+
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamSelf("cBookContent") ||
+		!L.CheckParamEnd(2)
+	)
+	{
+		return 0;
+	}
+
+	cBookContent * BookContent = nullptr;
+	L.GetStackValue(1, BookContent);
+	L.Push(BookContent->GetPages());
+	return 1;
+}
+
+
+
+
+
+static int tolua_cBookContent_SetPages(lua_State * tolua_S)
+{
+	// cBookContent::SetPages(table)
+
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamSelf("cBookContent") ||
+		!L.CheckParamTable(2) ||
+		!L.CheckParamEnd(3)
+	)
+	{
+		return 0;
+	}
+
+	cBookContent * BookContent = nullptr;
+	cLuaState::cStackTablePtr Pages;
+	L.GetStackValues(1, BookContent, Pages);
+	Pages->ForEachArrayElement([=](cLuaState & a_LuaState, int a_Index) -> bool
+		{
+			AString Page;
+			a_LuaState.GetStackValue(-1, Page);
+			BookContent->AddPage(Page);
+			return false;
+		}
+	);
```

Comment:
This seems to add pages, instead of setting them. So calling this multiple times will not replace, but instead append. Intended?